packages/services-soap/src/mode/seed.ts

```diff
@@ -64,9 +72,18 @@ export class SoapClientSupplierSeed extends SoapClientSupplierBase {
         this.testScope
       )
       try {
-        const resp = await fnc(args)
-        seed.set(resp)
-        return resp
+        try {
+          if (seed.expired) {
+            throw new Error()
+          }
+          const response = seed.get()
+          handleSeedError(response)
+          return response
+        } catch {
+          const response = await fnc(args)
+          seed.set(response)
+          return response
+        }
       } catch (e) {
         this.handleError(seed, e)
       }
```

Comment:
🎄 

Can this be done with a single try catch?

Reply:
no, but it can be refactored in a separate function, will be changed in refactoring